### PR TITLE
bug fix #1041

### LINKED ISSUE #1041: Clinical filter not working properly
Is clinical filter not working properly after the latest update? There are loads of intronic variants after filtering...

### PR DESCRIPTION
**Pull request by northwestwitch**
New release with bug fix #1041. Got rid of some javascript code and added items to the python form object instead!

### CODE REVIEW

**Comment by northwestwitch** [comment]
@moonso could you test it to see if it's fixed?

File: scout/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '4.1.4'
```
Comment (moonso) [inline scout/__version__.py]: Please don't update version in pull requests. I will make a release from master when it is merged

Comment (northwestwitch) [reply scout/__version__.py]: Ok!

**Comment by moonso** [comment]
👍

**Pull request merged**